test/bettingTest.js

```diff
@@ -4,17 +4,29 @@ contract("lupi", function(accounts) {
     var instance, ownerAddress;
     var gasUseLog = new Array();
     var salt = 0xdb8780d713083a9addb6494cfc767d6ef4b1358315737e06bbb7fd84cc493d1c;
+    var requiredBetAmount = 1000000000000000000;
+    var ticketCountLimit = 2;
+    var feePt = 10000;
 
     before(function(done) {
-        lupi.deployed()
+        lupi.new(requiredBetAmount, ticketCountLimit, feePt)
         .then( contractInstance => {
             instance = contractInstance;
             return instance.owner();
         }).then( ownerRes => {
             ownerAddress = ownerRes;
             done();
         });
-    }) // before()
+    }); // before()
+
+    it('contract should be set with initial parameters', function(done) {
+        return instance.getRoundInfo()
+        .then( res => {
+            assert.equal(res[0], 0, "state should be 'Betting' (0)");
+            assert(false, "TODO: assert other round params with bignumber compare");
+            done();
+        });
+    });
 
     it('should be possible to seal a bet for myself', function() {
         var sealedBet = 0x8eee4611778dd1659049d56a6f0ead527aad090d423fdc19c1407f586aa5ebff;
```

Comment:
1) testrpc runs with deterministic addresses (see runtestrpc.sh)
2) tests within one .js work on the same chain (ie. a consecutive it functions can see the changes of the next, assuming they run in sync)
so I still don't get why is it :)